modules/tinymce/src/plugins/wordcount/main/ts/core/Count.ts

```diff
@@ -14,11 +14,16 @@ import { getText } from './GetText';
 
 export type Counter = (node: Node, schema: Schema) => number;
 
+const removeZwsp = (text: string) =>
+  text.replace(/\u200B/g, '');
+
 const strLen = (str: string): number =>
   str.replace(/[\uD800-\uDBFF][\uDC00-\uDFFF]/g, '_').length;
 
 const countWords: Counter = (node: Node, schema: Schema): number => {
-  const text = getText(node, schema).join('\n');
+  // TINY-7484: The grapheme word boundary logic used by Polaris states a ZWSP (\u200B) should be treated as
+  // a word boundary, however word counting normally does not consider it as anything so we strip it out. 
```

Comment:
```suggestion
  // a word boundary, however word counting normally does not consider it as anything so we strip it out.
```
Accidentally left a space here sorry which is causing linter errors :cry: